android/java/org/chromium/chrome/browser/BraveRewardsPublisher.java

```diff
@@ -13,10 +13,11 @@
     //PublisherStatus @
     //vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom
     @Retention(RetentionPolicy.SOURCE)
-    @IntDef({NOT_VERIFIED, CONNECTED, UPHOLD_VERIFIED, BITFLYER_VERIFIED})
+    @IntDef({NOT_VERIFIED, CONNECTED, UPHOLD_VERIFIED, BITFLYER_VERIFIED, GEMINI_VERIFIED})
     public @interface PublisherStatus {}
     public static final int NOT_VERIFIED = 0;
     public static final int CONNECTED = 1;
     public static final int UPHOLD_VERIFIED = 2;
     public static final int BITFLYER_VERIFIED = 3;
+    public static final int GEMINI_VERIFIED = 4;
```

Comment:
the values are duplicated all over the place and it's totally unnecessary because they are defined in mojo which can be used in cpp, java and js. It's even done in two different places in ts. We have to stop copying code like this, it's not safe

Reply:
Actually it's even worse because it's defined in mojo and protobuf

Reply:
@jumde do you have a follow-up issue for this or is it part of the enum follow-up?

Reply:
New issue here: https://github.com/brave/brave-browser/issues/16866